source/plugins/go/src/extension/extension.go

```diff
@@ -79,7 +94,7 @@ func getExtensionSettings() (map[string]map[string]interface{}, error) {
 	}
 	for _, extensionConfig := range extensionConfigs {
 		extensionSettingsItr := extensionConfig.ExtensionSettings
-		if extensionSettingsItr != nil && len(extensionSettingsItr) > 0 {
+		if len(extensionSettingsItr) > 0 {

```

Comment:
why the nil check being removed?

Reply:
followed the linter: https://staticcheck.io/docs/checks#S1009